src/Doctrine/EntityRegenerator.php

```diff
@@ -105,7 +105,7 @@ public function regenerateEntities(string $classOrNamespace): void
                     continue;
                 }
 
-                $manipulator->addEntityField($fieldName, $mapping);
+                $manipulator->addEntityField($fieldName, $mapping, [], []);
```

Comment:
revert, `[]` is already the default value for the 2 last args

Reply:
done